algos/src/sample_rate_conversion.rst

```diff
@@ -75,139 +75,138 @@ scripts print intermediate information about progress).
+sample rate x1000, and stop band start relative to sample rate x1000. Many
+of the conversions are reused for other rates combinations with the same
 fractions.
 
 The vectors src_in_fs and src_out_fs list supported input and output
 rates. The arrays of structs src_table1 and src_table2 refer to the
 FIR filters coefficients used for the rates matrix. A special single
-tap FIR with coefficient 1.0 (Q2.30) is used when filtering is not
-needed. E.g. when input and output rate are equal or if a SRC stage is
+tap FIR with a coefficient of 1.0 (Q2.30) is used when filtering is not
+needed such as when the input and output rates are equal or if a SRC stage is
 not used.
 
 Coefficient precision
 =====================
 
-The coefficients can be generated as int16, int24, int32 or float
-type. The type is the 3rd argument for src_2stage(in_rates, out_rates,
-ctype) function call. It defaults to ‘int16’ that is the least memory
-consuming type that provides the minimum quality.  The 16 bit
-coefficients may achieve near up to 80-90 dB stopband that will give
-“near CD quality” conversion. The int32 and float type are capable to
-provide “CD quality” and better with a higher filter spec that is
+The coefficients can be generated as int16, int24, int32, or float
+type. The type is the 3rd argument for the src_2stage (in_rates, out_rates,
+ctype) function call. It defaults to ‘int16’, which is the least memory-consuming type that provides the minimum quality. The 16-bit
+coefficients may achieve near up to a 80-90 dB stopband that will give a
+“near CD quality” conversion. The int32 and float type are capable of
+providing “CD quality” and better with a higher filter spec that is
 explained later.
 
-The SRC component is using depending on build a "tiny" int16 or "std"
-int32 coefficient set. The testbench and FW build with xtensa compiler
-defaults to 32 bit coefficients. The gcc build build for firmware
-uses 16 bit coefficients. The scripts those were use to generated them
-are src_tiny_int16.m and src_std_int32.m.
+The type of SRC component used depends on whether you are building a "tiny"
+int16 or "std" int32 coefficient set. The testbench and FW build with xtensa
+compiler defaults to 32 bit coefficients. The gcc build for firmware uses 16
+bit coefficients. The scripts those were use to generated them are
+src_tiny_int16.m and src_std_int32.m.
 
 Exclusion of non-needed conversions
 ===================================
 
-If from previous call conversion set there would be no need for
-converting from 32 to 44.1 kHz it can be achieved by adding a matrix
-with zero in the place of the non-wanted conversion. This will help
-save memory that is needed to store the conversion coefficients.
+If previous call conversion sets had no need for converting from 32 to 44.1
+kHz, it can be achieved by adding a matrix with zero in the place of the
+non-wanted conversion. This will help save memory that is needed to store
+the conversion coefficients.
```

Comment:
@singalsu Please review this paragraph (beginning at line 161).

Reply:
There's more my mess left. I'd change also "If previous call conversion sets had no need..." into "If in the previous example set there would be no need..."